debezium-connector-oracle/src/main/java/io/debezium/connector/oracle/OracleConnectorConfig.java

```diff
@@ -285,7 +285,13 @@
     public static final Field LOG_MINING_BUFFER_TYPE = Field.create("log.mining.buffer.type")
             .withDisplayName("Controls which buffer type implementation to be used")
             .withEnum(LogMiningBufferType.class, LogMiningBufferType.MEMORY)
-            .withImportance(Importance.LOW);
+            .withImportance(Importance.LOW)
+            .withDescription("The buffer type controls how the connector manages buffering transaction data.\n" +
+                    "\n" +
```

Comment:
Let's use `System.lineSeparator()` instead, as this would not work as it should on Windows.